worker-workflow/src/test/resources/workflow-worker-test/sample-workflow.yaml

```diff
@@ -43,5 +43,8 @@ actions:
           }
   - name: action_2
     conditionFunction: function (document) { return fieldExists(document, 'field-should-exist');}
+    customData:
+      literalWithDoubleQuotes_1: "\"literalExample_1\""
+      literalWithDoubleQuotes_2: '"literalExample_2"'
```

Comment:
There's a third variant 
literalWithDoubleQuotes_3: "'literalExample_2''"

Reply:
HI Andy,
The previous action already had this variation -- valueFromLiteral: "'literalExample'" in line 34. So didnt add for this action.
But will add this as well to action_2 just so all variants are in one action.